spec/spec_helper.rb

```diff
@@ -1,6 +1,10 @@
 require 'simplecov'
 require 'simplecov-console'
 
+require 'database_connection'
+
+DatabaseConnection.connect('shop_manager_database')
```

Comment:
It would be beneficial to have a test database to run the tests on, this can prevent the data in the production database from being reset each time tests are run